pkg/buildpod/generator_test.go

```diff
@@ -301,13 +281,25 @@ func testGenerator(t *testing.T, when spec.G, it spec.S) {
 		})
 
 		it("passes in k8s service bindings if present", func() {
+			// Create provisioned service here b/c client wouldn't initialize with it
```

Comment:
Could you explain, What happened here? Did it not work to initialize with the fakeDynamicClient?

Reply:
That's right. For some reason, Tyler and I were unable to use the fakeDynamicClient to create the provisioned service under test.  At the moment, it's not clear to me why the fakeDynamicClient failed to meet our needs. More investigation will be done today.

Reply:
maybe we should go over this